tests/Knp/Menu/Tests/MenuItemTreeTest.php

```diff
@@ -13,11 +13,11 @@ final class MenuItemTreeTest extends MenuTestCase
 {
     public function testSampleTreeIntegrity(): void
     {
-        $this->assertCount(2, $this->menu);
-        $this->assertCount(3, $this->menu['Parent 1']);
-        $this->assertCount(1, $this->menu['Parent 2']);
-        $this->assertCount(1, $this->menu['Parent 2']['Child 4']);
-        $this->assertEquals('Grandchild 1', $this->menu['Parent 2']['Child 4']['Grandchild 1']->getName());
+        $this->assertCount(2, $this->menu->getChildren());
+        $this->assertCount(3, $this->pt1->getChildren());
+        $this->assertCount(1, $this->pt2->getChildren());
+        $this->assertCount(1, $this->ch4->getChildren());
+        $this->assertEquals('Grandchild 1', $this->gc1->getName());
```

Comment:
This is totally changing the assertions being done. It does not test the integrity of the ArrayAccess anymore.